build.gradle

```diff
@@ -149,11 +148,6 @@ ext {
         // For this repo we'll include everything that's not a sample or testonly module.
         return !subProj.name.startsWith("sample") && !subProj.getName().startsWith("testonly")
     }
-
-    // BINTRAY STUFF
-    bintrayUser = project.hasProperty('bintrayUser') ? property('bintrayUser') : 'UNDEFINED'
```

Comment:
The bintray publishing stuff will be replaced with maven central (sonatype/OSSRH) publishing in a later PR.